package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -406,7 +407,7 @@ def __init__(self, universe, selection1='protein', selection2='all', selection1_
                  distance=3.0, angle=120.0,
                  forcefield='CHARMM27', donors=None, acceptors=None,
                  start=None, stop=None, step=None,
-                 debug=None, detect_hydrogens='distance', verbose=None):
+                 debug=None, detect_hydrogens='distance', verbose=None,pbc=True):
```

Comment:
Include a description of this new optional argument in the the `Parameters` section of the class' docstring

Reply:
The default for pbc will need to be False otherwise this class will behave differently (with defaults) before and after this patch